docs/example-scenario/data/data-contextualization-based-on-azure-sql-graph-content.md

```diff
@@ -70,13 +70,13 @@ The solution assumes that a graph has already been created in a graph database.
 
 The solution uses Azure Databricks for the data contextualization process.
 
-### Graph database
+### Graph data
```

Comment:
@kenakamu I prefer to use Graph database as this section talks about the options of graph databases to store the graph data.

Reply:
okay